portfolio/src/main/webapp/style.css

```diff
@@ -2,49 +2,131 @@
 @include button.core-styles;
 @use "@material/card";
 @include card.core-styles;
+@import url('https://fonts.googleapis.com/css?family=Lato:100,100i,300,300i,400,400i,700,700i,900,900i');
 
+html {
+   background-size: cover;
+   background-color: #6495ED;
+   height:100%;
+}
+
+/*
+ * Credit to codepen: https://codepen.io/valentin/pen/fkdCG
+*/
 body {
-  background-color: #8EC5FC;
-  background-image: linear-gradient(62deg, #8EC5FC 0%, #E0C3FC 100%);
+  animation: wind 20s linear infinite;
+  background-color: #808080;
+  background: url(https://static.radulescu.me/examples/clouds/clouds1000.png);
+  background: url(https://static.radulescu.me/examples/clouds/clouds1000.png) 0 200px,
+	      url(https://static.radulescu.me/examples/clouds/clouds1200_1.png) 0 300px,
+	      url(https://static.radulescu.me/examples/clouds/clouds1000_blur3.png) 100px 250px;
   color: white;
-  margin: auto;
-  padding: 1%;
+  margin: 0;
+  min-height: 100%;
+  padding: 0;
+  position: relative;
+}
+
+@keyframes wind {
+   0% {
+	background-position: 0 200px, 0 300px, 100px 250px;
+      }
+   100% {
+	  background-position: 1000px 200px, 1200px 300px, 1100px 250px;
+	}
 }
 
 #intro-content {
   margin-left: auto;
   margin-right: auto;
-  text-align:center;
-  width: 650px;
+  text-align: center;
+  width: 100%;
+}
+
+#intro-button-one {
+  margin: auto;
+}
+
+#profile-wrapper {
+   margin: auto;
+   padding-top: 15px;
 }
 
 #basic-profile {
+   background-color: rgba(128,128,128,.6);
    border: 1px solid black;
-   padding: 10px;
+   font-weight: bold;
+   padding-top: 10px;
+   padding-left: 20px;
+   padding-right: 20px;
+   padding-bottom: 5px;
+}
+
+#intro-button-two {
+   margin: auto;
 }
 
 #inner-content {
    border: 1px solid black;
    text-align: center;
 }
 
-#intro-buttons {
-  display: inline-block;
-  padding: 1%;
-}
-
 #intro {
   margin: auto;
   width: 300px;
 }
 
-img {
+#info {
+   border: 1px solid black;
+   margin-top: -50px;
+   margin-right: auto;
+   margin-left: auto;
+   text-align: center;
+   width: 250px;
+}
+
+#profile {
+  background: #CCC;
   border: 1px solid white;
   box-shadow: 0 19px 30px #0000004C, 0 15px 12px #00000038;
   height: 400px;
+  margin: 10px;
+  position: relative;
   width: 350px;
+  transform: rotateX(0) rotateY(0);
+  transform-style: preserve-3d;
+  transition-duration: 0.1s;
+  transition-timing-function: ease !important;
```

Comment:
Is this for the parallax? If so, remove it from this cl about formatting

Reply:
ah ok I do believe it is-I will remove it. Thanks!